app/decorators/manageiq/providers/automation_manager_decorator.rb

```diff
@@ -1,17 +1,35 @@
-module ManageIQ::Providers
-  class AutomationManagerDecorator < MiqDecorator
-    def self.fonticon
-      nil
-    end
+class ManageIQ::Providers::AutomationManagerDecorator < MiqDecorator
+  def self.fonticon
+    nil
+  end
+
+  def fileicon
+    "svg/vendor-#{image_name.downcase}.svg"
+  end
 
-    def fileicon
-      "svg/vendor-#{image_name.downcase}.svg"
-    end
+  def single_quad
+    {
+      :fileicon => fileicon
+    }
+  end
 
-    def single_quad
-      {
-        :fileicon => fileicon
-      }
-    end
+  def quadicon
+    icon = {
+      :top_left     => {
+        :text    => t = inventory_root_groups.size,
+        :tooltip => n_("%{number} Inventory Root Group", "%{number} Inventory Root Groups", t) % {:number => t}
+      },
+      :top_right    => {
+        :text    => t = configuration_scripts.size,
+        :tooltip => n_("%{number} Configuration Script", "%{number} Configuration Scripts", t) % {:number => t}
```

Comment:
I'm not sure about the tooltip of this one, in the issue @slemmartin mentioned `Ansible Tower templates` but the method name is `configuration_scripts`. What's the right name here?

Reply:
Ansible Tower templates is term used by UI now, see https://github.com/ManageIQ/manageiq-ui-classic/issues/5188. The name ConfigurationScript is inferred from base model and table name, but this table contains also playbooks which are not part of list in UI shown in screenshot)

Reply:
Okay, so what method(s) should I use on the EMS instance to get the count of the templates?

Reply:
looking into data, it seems that template consists from several playbooks, right?

Reply:
> Okay, so what method(s) should I use on the EMS instance to get the count of the templates?

Automation_manager > Ansible Tower > Explorer, tab Templates> choose some manager > number in pagination

Reply:
@slemrmartin by method I ment a ruby method :rofl:

Reply:
`configuration_scripts.count`